src/main/java/com/um/inventoryservice/DataLayer/Employee.java

```diff
@@ -0,0 +1,21 @@
+package com.um.inventoryservice.DataLayer;
+
+import lombok.*;
+import org.springframework.data.annotation.Id;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Employee {
+
+    @Id
+    private String id;
+
+    private String employeeId;
+    private String employeeName;
+    private String position;
```

Comment:
Is position where we distinguish between employee and admin?

Reply:
that should be it, but it could be also the positions of the different employees too.